.drone.yml

```diff
@@ -0,0 +1,120 @@
+---
+platform:
+  os: linux
+  arch: amd64
+
+# Begin
+clone:
+  git-clone:
+    commands: |
+      set -ex
+      git clone -b ${DRONE_TAG:-$DRONE_BRANCH} $DRONE_REMOTE_URL .
+      if [ x$DRONE_PULL_REQUEST != x ]; then
+          git fetch origin refs/pull/$DRONE_PULL_REQUEST/head
+          EMAIL=ci git merge --no-edit FETCH_HEAD
+      fi
+      git rev-parse HEAD
+    image: "casperlabs/buildenv:latest"
+
+kind: pipeline
+name: build-bors-merge
+
+steps:
+- name: lint-pr
+  commands:
+  - "make lint"
+  image: "casperlabs/buildenv:latest"
+  when:
+    event:
+    - pull_request
+
+- name: test-pr
+  commands:
+  - "rustup target add wasm32-unknown-unknown"
+  - "make test"
+  image: "casperlabs/buildenv:latest"
+  when:
+    event:
+    - pull_request
+    status:
+    - success
+  depends_on:
+  - lint-pr
+
+- name: test-push
+  commands:
+  - "rustup target add wasm32-unknown-unknown"
+  - "make test"
+  image: "casperlabs/buildenv:latest"
+  when:
+    event:
+    - push
+    status:
+    - success
+
+- name: lint-bors
+  commands:
+  - "make lint"
+  image: "casperlabs/buildenv:latest"
+  when:
+    branch:
+    - staging
+    - trying
+
+- name: test-bors
+  commands:
+  - "rustup target add wasm32-unknown-unknown"
+  - "make test"
+  image: "casperlabs/buildenv:latest"
+  when:
+    branch:
+    - staging
+    - trying
+  depends_on:
+  - lint-bors
+
+
+volumes:
+- name: docker_sock
+  host:
+    path: "/var/run/docker.sock"
+- name: temp
+  host:
+    path: "/tmp"
```

Comment:
this isn't being used from what i can see.

Reply:
isn't this for the `image` in steps ?

Reply:
this is for using host volume mounts. Theres no point to mount them if they are not being used.